framework/math/quadratures/angular/cylindrical_quadrature.cc

```diff
@@ -1,71 +1,65 @@
-#include "framework/math/quadratures/cylindrical_angular_quadrature.h"
-
+#include "framework/math/quadratures/angular/cylindrical_quadrature.h"
+#include "framework/logging/log.h"
+#include "framework/runtime.h"
 #include <algorithm>
 #include <limits>
 #include <numeric>
 
-#include "framework/runtime.h"
-#include "framework/logging/log.h"
-
 namespace opensn
 {
 
-CylindricalAngularQuadrature::CylindricalAngularQuadrature(const Quadrature& quad_polar,
-                                                           const Quadrature& quad_azimu,
-                                                           const bool verbose)
-  : CurvilinearAngularQuadrature()
+CylindricalQuadrature::CylindricalQuadrature(const SpatialQuadrature& quad_polar,
+                                             const SpatialQuadrature& quad_azimu,
```

Comment:
fishy name: spatial quadrature but it is for pola and azimu ??

Reply:
This used to be `Quadrature` which was the base class for the spatial quadratures. It's just more explicit now.

Reply:
The CylindricalQuadrature takes a Gauss-Legendre quadrature for `quad_polar` and either Gauss-Legendre or Gauss-Chebyshev quadrature for `quad_azimu`. Both of those are derived from `SpatialQuadrature`.

Reply:
I've created I a new base class for the Gauss quadratures (GaussQuadrature) so they no longer derive from SpatialQuadrature. I think this is a much cleaner implementation. In addition, LineQuadrature now has SpatialQuadrature as its base class instead of GaussLegendreQuadture.